forte/data/selector.py

```diff
@@ -59,19 +62,41 @@ class SinglePackSelector(Selector[MultiPack, DataPack]):
     This is the base class that select a DataPack from MultiPack.
     """
 
+    def __init__(self,
```

Comment:
Do we need to override the base `__init__()` here? Seems like it's not doing anything special.

Reply:
You are right. I'll remove it.